src/OpenTelemetry/.publicApi/net462/PublicAPI.Shipped.txt

```diff
@@ -1,131 +1,64 @@
 #nullable enable
-abstract OpenTelemetry.BaseExporter<T>.Export(in OpenTelemetry.Batch<T!> batch) -> OpenTelemetry.ExportResult
```

Comment:
how  come we are removing things from shipped.txt? Or is it just reordered/

Reply:
It's just sorted now. We had added the `~` characters manually before the stable release and the finalize build script sorts the text when run.

Reply:
It's still 381 lines: 351 that were already in Shipped.txt + 30 new lines from Unshipped.txt

Reply:
https://www.fuget.org/packages/OpenTelemetry/1.4.0-rc.4/lib/netstandard2.0/diff/1.3.2/ this gave neater view and confirms nothing was removed!